### PR TITLE
Update wp.tex

### PR DESCRIPTION
**Pull request by IlyaMoskva**
Issue #1  First version of full WP document.

### CODE REVIEW

**Comment by IlyaMoskva** [comment]
Written and tested with Overleaf https://www.overleaf.com/ as desktop can't push to github.

File: white-paper/wp.tex [hunk 1]
```diff
@@ -65,34 +65,177 @@
...
+
+Probability of any network issues be provided as settings for simulator.
+
+Frequency of all possible operations and average time for processing must be set as initial parameters.
+
+To generate graph the following settings are needed to be set:
+\begin{itemize}
+    \item Number of regions;
+    \item Number of nodes;
+    \item Number of repositories;
```
Comment (g4s8) [inline white-paper/wp.tex:130]: @IlyaMoskva I think we can abstract away of repositories and just call it as node state - this simulator can be potentially used not for git repository management only. Se the init script sets up initial state for node.

File: white-paper/wp.tex [hunk 2]
```diff
@@ -65,34 +65,177 @@
...
+        \item State (Active/Down)
+    \end{itemize}
+    Edges of graph are routes from one Node to another. They must have their own configuration properties:
+    \begin{itemize}
+        \item Throughput (max Mb/sec)
+        \item State (Active/Down)
+    \end{itemize}
+    Parameters for nodes and edges must be set with given distribution method. (Normal, random, etc)
+    
+    \item[Repository.]
```
Comment (g4s8) [inline white-paper/wp.tex:224]: @IlyaMoskva again, I'd call it node state

**Review by g4s8 — CHANGES_REQUESTED** [review]
@IlyaMoskva please see my comments

File: white-paper/wp.tex (hunk 1 above)
Comment (IlyaMoskva) [reply white-paper/wp.tex:130]: @g4s8 I meant here a number how many repositories Generator should create as entities before start Simulation. The value expected here is 1.000.000, for example. And that set of repos will be "placed" on nodes.

File: white-paper/wp.tex (hunk 2 above)
Comment (IlyaMoskva) [reply white-paper/wp.tex:224]: @g4s8 , I agree

**Comment by IlyaMoskva** [comment]
@g4s8 Please, review again.

**Review by g4s8 — APPROVED** [review]
@IlyaMoskva thanks

**Pull request merged**